### PR TITLE
feat(atomic,insight): update locales.json and insights.html to support viewed by customer

### PR DESCRIPTION
**Pull request by jfallaire**
Add **Viewed By Customer** support for hosted insight panel
Shows a badge when a customer has viewed the specific result

https://coveord.atlassian.net/browse/SVCINT-3062

image

### CODE REVIEW

**Review by mikegron — APPROVED** [review]
Quand meme pas pire je trouve, les gars de DX pourrons valider si le css est legit :p

File: packages/atomic/src/pages/examples/insights.html [hunk 1]
```diff
@@ -327,6 +327,28 @@
                   .result-root.image-large atomic-result-section-visual {
                     border-radius: var(--atomic-border-radius-xl);
                   }
+
+                  .result-root.with-sections.display-list.density-normal atomic-result-section-bottom-metadata {
+                    max-height: 4.5rem !important;
```
Comment (louis-bompart) [inline packages/atomic/src/pages/examples/insights.html:332]: `!important`, ark.

Comment (jfallaire) [reply packages/atomic/src/pages/examples/insights.html:332]: yes good point! thank you.

File: packages/atomic/src/locales.json [hunk 2]
```diff
@@ -5265,6 +5265,33 @@
     "zh-CN": "与本案有关的见解",
     "zh-TW": "與此案件相關的洞見"
   },
+  "viewed-by-customer": {
+    "en": "Viewed by customer",
+    "fr": "Vu par le client",
+    "cs": "Zobrazeno zákazníkem",
```
Comment (fpbrault) [inline packages/atomic/src/locales.json:5271]: I think this should be zákazníkovi (by the customer)
```suggestion
    "cs": "Zobrazeno zákazníkovi",
```

**Review by fpbrault — APPROVED** [review]

**Review by alexprudhomme — APPROVED** [review]
Will this ever be not internal ? If so, this is quite a lot of css rules for an implementer to write. Would it become a component one day ?

**Comment by mikegron** [comment]
> Will this ever be not internal ? If so, this is quite a lot of css rules for an implementer to write. Would it become a component one day ?

It will be used in integrations mostly.. Not sure if a component would be easier to maintain?

**Pull request merged**